src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/CSharpFormattingOptions2.cs

```diff
@@ -311,6 +415,32 @@ internal enum BinaryOperatorSpacingOptionsInternal
     Remove = 2
 }
 
+internal enum ParameterWrappingOptionsInternal
+{
+    DoNotWrap,
+    WrapLongParameters,
+    WrapEveryParameter
+}
```

Comment:
can you doc all of these.  we shoudl also doc how these interact.  It would be good to have an example for all 12 permutations of these parameter options.

Note: we will need this for argument lists as well.